lib/matplotlib/_constrained_layout.py

```diff
@@ -272,7 +272,11 @@ def _make_layout_margins(ax, renderer, h_pad, w_pad):
     invTransFig = fig.transFigure.inverted().transform_bbox
     pos = ax.get_position(original=True)
     tightbbox = ax.get_tightbbox(renderer=renderer)
-    bbox = invTransFig(tightbbox)
+    if tightbbox is None:
```

Comment:
Just:
~~~
bbox = pos if tightbbox is None else invTransFig(tightbbox)
~~~

Reply:
I find those really hard to read a lot of the time.

Reply:
Actually, when reading a function for a basic understanding what it does I find this more easy.

It's just one line saying: "Sets `bbox` depending on some condition". The if/else is already more low level.

But not fighting over this.